.eslintrc.js

```diff
@@ -1,6 +1,7 @@
 module.exports = {
   extends: ['algolia/react', 'algolia/jest'],
   rules: {
+    'max-params': ['error', 7],
```

Comment:
YOLO! For a next major I bet you would want to move over to an object param maybe?

Reply:
Clearly... I tried to find a path to easily migrate on named parameters (with `fn.length`) but without renaming the function I didn't found an easy way.